common/lib/dependabot/pull_request_creator/message_builder.rb

```diff
@@ -327,6 +327,10 @@ def source_url(dependency)
         metadata_finder(dependency).source_url
       end
 
+      def homepage_url(dependency)
+        metadata_finder(dependency).homepage_url
+      end
+
```

Comment:
Does this warrant adding a test against removal of the method in the future?

Reply:
👍🏻  I removed some self-stubbing on the unit-under test which was bypassing over these method calls.